Build Error  error: static assertion failed: Failed to reserve enough space in fc::fwd<T,S>

**Instructions**
Please include a detailed Title above. Next, please complete the following sections below:
* Build Error
In file included from /root/bitshares-core/libraries/fc/src/network/tcp_socket.cpp:4:0:
/root/bitshares-core/libraries/fc/include/fc/fwd_impl.hpp: In instantiation of ‘void fc::check_size() [with long unsigned int RequiredSize = 96ul; long unsigned int ProvidedSize = 84ul]’:
/root/bitshares-core/libraries/fc/include/fc/fwd_impl.hpp:85:43:   required from ‘fc::fwd<T, S, Align>::fwd() [with T = fc::tcp_socket::impl; unsigned int S = 84u; Align = double]’
/root/bitshares-core/libraries/fc/src/network/tcp_socket.cpp:91:26:   required from here
/root/bitshares-core/libraries/fc/include/fc/fwd_impl.hpp:60:25: error: static assertion failed: Failed to reserve enough space in fc::fwd<T,S>
     void check_size() { static_assert( (ProvidedSize >= RequiredSize), "Failed to reserve enough space in fc::fwd<T,S>" ); }
                         ^
libraries/fc/CMakeFiles/fc.dir/build.make:816: recipe for target 'libraries/fc/CMakeFiles/fc.dir/src/network/tcp_socket.cpp.o' failed
make[2]: *** [libraries/fc/CMakeFiles/fc.dir/src/network/tcp_socket.cpp.o] Error 1
CMakeFiles/Makefile2:183: recipe for target 'libraries/fc/CMakeFiles/fc.dir/all' failed
make[1]: *** [libraries/fc/CMakeFiles/fc.dir/all] Error 2
Makefile:129: recipe for target 'all' failed 


**Build Error Description**
error: static assertion failed: Failed to reserve enough space in fc::fwd<T,S>
 my os is ubuntu 16.04
when build test-3.3.0 and test-3.3.1 , the same build error occured.

help!

--- a/include/fc/network/tcp_socket.hpp
+++ b/include/fc/network/tcp_socket.hpp
@@ -53,7 +53,7 @@ namespace fc {
       #ifdef _WIN64
       fc::fwd<impl,0xa8> my;
       #else
-      fc::fwd<impl,0x54> my;
+      fc::fwd<impl,0x60> my;
       #endif
   };
   typedef std::shared_ptr<tcp_socket> tcp_socket_ptr;

